diff.py

```diff
@@ -291,6 +296,32 @@ def search_map_file(fn_name):
     return None, None
 
 
+def dump_elf():
+    if not baseimg or not myimg:
+        fail("Missing myimg/baseimg in config.")
+    if base_shift:
+        fail("--base-shift not compatible with -e")
+    if args.end is None:
+        fail("The end address is required.")
+
+    start_addr = eval_int(args.start)
+    end_addr = eval_int(args.end)
```

Comment:
these should take error messages, otherwise eval_int returns None if it fails to parse

Reply:
could be worth doing
```py
    if args.end is not None:
        end_addr = eval_int(args.end, "End address must be an integer expression.")
    else:
        end_addr = start_addr + MAX_FUNCTION_SIZE_BYTES
```
just for consistency